cmd/keymasterd/main.go

```diff
@@ -489,6 +490,14 @@ func (state *RuntimeState) writeFailureResponse(w http.ResponseWriter, r *http.R
 	}
 }
 
+func setSecurityHeaders(w http.ResponseWriter) {
+	//all common security headers go here
+	w.Header().Set("Strict-Transport-Security", "max-age=31536")
```

Comment:
Just something to consider: would we want to add `includeSubDomains` to the HSTS parameters too? Or are you sure there won't be any subdomains? :) Either way can't think of a way it would hurt if added.

Reply:
Include subdomains can be used to fingerprint users. Actually It makes sense here to increase the max-age of the HSTS header.